packages/contracts/hardhat.config.ts

```diff
@@ -58,7 +58,7 @@ module.exports = {
       tags: ["test", "demo"],
       chainId: 5,
       accounts: real_accounts,
-      gatewayurl,
+      gatewayurl: "http://localhost:8080/{sender}/{data}.json",
```

Comment:
Maybe use a `localGatewayURL` variable to store it?

Reply:
Ok! I'll try it!